controllers/machine_controller_noderef.go

```diff
@@ -48,7 +48,7 @@ func (r *MachineReconciler) reconcileNodeRef(ctx context.Context, cluster *v1alp
 
 	// Check that Cluster isn't nil.
 	if cluster == nil {
-		klog.Warningf("Machine %q in namespace %q doesn't have a linked cluster, won't assign NodeRef", machine.Name, machine.Namespace)
+		klog.V(2).Infof("Machine %q in namespace %q doesn't have a linked cluster, won't assign NodeRef", machine.Name, machine.Namespace)
```

Comment:
Shouldn't we be avoiding `.V()`? I know @chuckha has thoughts here :)

Reply:
This is just a stopgap until we remove raw klog calls and solve this otherwise

Reply:
(it's only verbose in logs)

Reply:
anything goes with global logging :D